Zend/tests/gh14456.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+GH-14456: Attempting to initialize class with private constructor calls destructor
+--FILE--
+<?php
+
+class PrivateUser {
+    private function __construct() {
+        return new stdClass();
```

Comment:
It doesn't seem like this is relevant to the test?

Reply:
I fixed it while merging